### DISCUSSION TITLE
Not able to open the Cluster Dashboard.

### DISCUSSION THREAD

**Discussion by ChristianMe96** [rancher-sandbox/rancher-desktop#5326] [Q&A] [answered]
Hello,

I'm pretty new to all this and just installed rancher-desktop the first time, sadly, the cluster dashboard won't open, and I just don't get the reason. I have a fresh installation, my config only includes the local cluster of rancher-desktop.

The dashboardServer.log File looks like this:
`2023-08-11T11:32:35.779Z: [HPM] Proxy created: /  -> https://127.0.0.1:9443
2023-08-11T11:32:35.779Z: [HPM] Proxy created: /  -> https://127.0.0.1:9443
2023-08-11T11:32:35.781Z: [HPM] Proxy created: /  -> https://127.0.0.1:9443
2023-08-11T11:34:59.349Z: Proxy Error: Error: connect ECONNREFUSED 127.0.0.1:9443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1187:16) {
  errno: -61,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9443
}
2023-08-11T11:34:59.350Z: [HPM] Error occurred while proxying request 127.0.0.1:6120/api/v1/namespaces/cattle-ui-plugin-system/services/http:ui-plugin-operator:80/proxy/index.json to https://127.0.0.1:9443/ [ECONNREFUSED] (https://nodejs.org/api/errors.html#errors_common_system_errors)`

local config:
`apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: DATA+OMITTED
    server: https://127.0.0.1:6443
  name: rancher-desktop
contexts:
- context:
    cluster: rancher-desktop
    user: rancher-desktop
  name: rancher-desktop
current-context: rancher-desktop
kind: Config
preferences: {}
users:
- name: rancher-desktop
  user:
    client-certificate-data: DATA+OMITTED
    client-key-data: DATA+OMITTED`

Installed Version: Rancher.Desktop-1.9.1-mac.aarch64.zip
System: Mac with M1 Prozessor

**Comment by adamkpickering** [comment]
Have you tried a factory reset? This can sometimes resolve problems.

Do you have any sort of IT management software (things like jamf) installed on your device? Sometimes this type of software causes problems.

Otherwise I'd recommend opening an issue so that we can gather more information and maybe solve your problem.

**Reply by ChristianMe96** [reply]
Hey,

Thanks for the quick reply.
I did try the factory reset which didn't work. But I also do have jamf installed on my device, is there like a workaround or do I just have to live with it. 😅

**Reply by adamkpickering** [reply, marked as answer]
I looked to see if the port the dashboard listens on (9443) is configurable and it is not. So unfortunately I don't have any ideas for a workaround.

If you absolutely need this functionality, I think your best bet is to take it up with IT. Maybe an exception can be made for this port?

**Reply by ChristianMe96** [reply]
I found out that it's not related to jamf, since it also occurred on my Windows Maschine. After looking in the steve.log file, i found out that port 9080 was already in use and didn't start because of that. The Port was used by Logitech Hub (don't ask me why they use this port). 😅

**Comment by amardeep2006** [comment]
I faced same issue due to nahimic service blocking 9080 port and was solved after disabling it.

**Comment by mikeseese** [comment]
For people finding via Google, Logitech G Hub also listens on 9080. Uninstalling it got the dashboard working for me after a "Reset Kubernetes".
